common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OnPageCommitVisibleCallback.java

```diff
@@ -22,6 +22,6 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.ui.webview;
 
-public interface OnPageLoadedCallback {
-    void onPageLoaded();
+public interface OnPageCommitVisibleCallback {
```

Comment:
This is an API breaking change for common - we should avoid this if we can.

Let's instead:
- Add a new interface
- Add a new ctor to `OAuth2WebViewClient` to accept that interface
- Switch relevant code to call new ctor

Reply:
OK. Maybe let me just change the caller method.

Reply:
Thanks, added new `OnPageCommitVisibleCallback` and new ctor.